src/current/_includes/v22.2/app/java-tls-note.md

```diff
@@ -1,4 +1,3 @@
-{{site.data.alerts.callout_danger}}
```

Comment:
It was a danger sign because java versions with this bug were common when this note was written, so it was common for java clients to be unable to connect to CockroachDB and this needed to be emphasized. Now that the buggy versions of java are less prevalent we can remove the danger sign.